src/core/backbone_router/multicast_listeners_table.cpp

```diff
@@ -308,6 +308,20 @@ Error MulticastListenersTable::GetNext(Listener::Iterator &aIterator, Listener::
     return error;
 }
 
+bool MulticastListenersTable::HasListener(const Ip6::Address &aAddress)
+{
+    for (uint16_t i = 0; i < mNumValidListeners; i++)
+    {
+        Listener &listener = mListeners[i];
```

Comment:
- Probably this variable is unnecessary.
- In a future PR, should we use `Array` to store listeners so that the implementation is simpler?

Reply:
Removed.
I'll have a future PR to use Array for the listeners.